DbmProgram/app/pages/messagerouterdemo.html

```diff
@@ -0,0 +1,261 @@
+            border-bottom: 1px solid black;
+            border-collapse: collapse;
+            vertical-align: middle;
+        }
+
+        .selected {
+            background-color: Highlight;
+            color: black;
+        }
+
+        .tool {
+            position: relative;
+            /*width: 400px;*/
+            cursor: pointer;
+        }
+
+            .tool .tooltiptext {
+                visibility: hidden;
+                /*width: 400px;*/
+                background-color: black;
+                color: #fff;
+                text-align: center;
+                border-radius: 6px;
+                padding: 5px 0;
+                /* Position the tooltip */
+                position: absolute;
+                z-index: 1;
+                top: 100%;
+                left: 0%;
+                /*margin-left: -60px;*/
+            }
+
+            .tool:hover .tooltiptext {
+                visibility: visible;
+            }
+    </style>
+
+    <script type="text/javascript">
+
+
+        $(function () {
+            var source = $("#UnexecutedScripts").html();
+            var template = Handlebars.compile(source);
+
+
+            //$('#buttonMessageRouterRun').on('click', showAll);
+            $(function () {
+
+                $('#ScriptRouterResult').html('');
+                $('#result').empty();
+
+                var request = {
+                    "method": "GET",
+                    "url": "/uicontroller/getUnexecuted",
+                    "parameters": null,
+                    "postData": null,
+                };
+                window.cefQuery({
+                    request: JSON.stringify(request),
+                    onSuccess: function (response) {
+                        ScriptRouterResult(response);
+                    }, onFailure: function (err, msg) {
+                        console.log(err, msg);
+                    }
+                });
+
+                var request = {
+                    "method": "GET",
+                    "url": "/uicontroller/getAll",
+                    "parameters": null,
+                    "postData": null,
+                };
+                window.cefQuery({
+                    request: JSON.stringify(request),
+                    onSuccess: function (response) {
+                        messageRouterResult(response);
+                    }, onFailure: function (err, msg) {
+                        console.log(err, msg);
+                    }
+                });
+            });
+
+
+
+            function ScriptRouterResult(res) {
+
+                var jsonData = JSON.parse(res);
+
+                var data = jsonData.Data
+                if (data.length > 0) {
+                    var html = template(data);
+                    $('#ScriptRouterResult').html(html);
+                }
+
+                unexecutedScriptNumRouter('New: ' + data.length + ' Scripts ');
+
+                $('#scriptTable tbody tr td .view').click(function () {
```

Comment:
this is a terrible selector. You can add any attribute to an element. Why make this so complex?